spyder/plugins/ipythonconsole/utils/kernelspec.py

```diff
@@ -62,13 +94,43 @@ def argv(self):
                 pyexec = pyexec_w
 
         # Command used to start kernels
-        kernel_cmd = [
-            pyexec,
-            '-m',
-            'spyder_kernels.console',
-            '-f',
-            '{connection_file}'
-        ]
+        if is_different_interpreter and is_conda():
+            # If this is a conda environment we need to call an intermediate
+            # activation script to correctly activate the spyder-kernel
+
+            # If changes are needed on this section make sure you also update
+            # the activation scripts at spyder.plugins.ipythonconsole.scripts
```

Comment:
Please change the dots here for slashes:

    spyder/plugins/ipythonconsole/scripts